### PR TITLE
Milestone 6: The Next Web

### PR DESCRIPTION
**Pull request by MohameDawood**
This page is a clone of a The Next Web page made for the Milestone 6 project of Microverse & Odin Project.

### CODE REVIEW

**Comment by MohameDawood** [comment]
I couldn't configure the linter to check the SCSS so I hope you lead me here

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1591 @@
...
+    <link rel="stylesheet" href="main.css" />
+    <title>TNW</title>
+    <script
+      src="https://kit.fontawesome.com/63c1245c39.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <!-- Navigator -->
+    <nav class="nav nav-m-b">
```
Comment (tirthajyoti-ghosh) [inline index.html:15]: The top section(the one with the dark background) of the navbar is missing in your clone. Please add that.

| Source Page	| Clone Page |
|:---:|:---:|
|Annotation 2020-07-10 024642|image|

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1591 @@
...
+          </li>
+        </ul>
+      </div>
+    </nav>
+
+    <!-- Main Conetent -->
+    <main>
+      <div class="slider__section">
+        <!-- Start The slider Block -->
+        <article class="slider">
```
Comment (tirthajyoti-ghosh) [inline index.html:69]: Please add the TNW logo.

| Source Page	| Clone Page |
|:---:|:---:|
|image| image|

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1591 @@
...
+                  </ul>
+                </div>
+              </li>
+              <!-- End Latest News Artile Block -->
+            </ul>
+          </section>
+          <!-- End The Latest News Section -->
+
+          <!-- Start The Funding Section -->
+          <section class="funding s-m-b">
```
Comment (tirthajyoti-ghosh) [inline index.html:375]: The latest funding section should contain 4 articles in desktop mode. Please add one more.

| Source Page	| Clone Page |
|:---:|:---:|
|image|image|

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1591 @@
...
+          <div class="topic-section s-m-b row">
+            <!-- Start The topic Section -->
+            <section class="topic col-m--2">
+              <!-- Topic Heading -->
+              <h2 class="topic__heading--main second-heading topic-m-b">
+                <a href="#">Hard Fork</a>
+              </h2>
+              <ul class="topic__list">
+                <li class="topic__item">
+                  <article class="topic__article--frist">
```
Comment (tirthajyoti-ghosh) [inline index.html:497]: Please reduce the space here 👇
Annotation 2020-07-10 025647

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1591 @@
...
+              <!-- End Latest Deal Artile Block -->
+            </ul>
+          </section>
+        </div>
+      </div>
+    </main>
+
+    <!-- Start Footer -->
+    <footer class="footer">
+      <div class="link-box">
```
Comment (tirthajyoti-ghosh) [inline index.html:1526]: The footer should not be changing its style throughout the breakpoints. It should be maintaining the same style in all screen sizes.

| Source Page	| Clone Page |
|:---:|:---:|
|image|Annotation 2020-07-10 025946|

File: index.html (hunk 1 above)
Comment (tirthajyoti-ghosh) [inline index.html:15]: In the mobile view, the hamburger icon is missing from the navbar.

| Source Page	| Clone Page |
|:---:|:---:|
|Annotation 2020-07-10 030306|image|

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# You have done an incredible job MAKING THIS PROJECT 👏💪! 

## However, there are some changes still required. 

### Please go through the above comments and make the changes. 

Please pay attention to detail. It's going to help you in the long run. Also, pay attention to the footer. It shouldn't be changing styles throughout the breakpoints.

Request another code review when you are done.

Happy coding 👨‍💻

**Pull request closed without merging**